### PR TITLE
Forward port of Batch Revoke Entitlements to 0.9.51

### PR DESCRIPTION
**Pull request by nguyenfilip**

### CODE REVIEW

**Comment by nguyenfilip** [comment]
It will probably be necessary to backport 0988fdac3c588b479aa930938bee7eb3842fe15f into this. I will wait for Jenkins to try running spec tests

File: server/src/test/java/org/candlepin/controller/PoolManagerFunctionalTest.java [hunk 1]
```diff
@@ -190,6 +190,21 @@ public void testQuantityCheck() throws Exception {
     }
 
     @Test
+    public void testDeletePool() throws Exception {
+        Pool pool = createPoolAndSub(o, socketLimitedProduct, 100L,
+            TestUtil.createDate(2000, 3, 2), TestUtil.createDate(2050, 3, 2));
+        poolCurator.create(pool);
+        List<Pool> pools = poolCurator.listByOwner(o);
+        assertEquals(5, poolCurator.listByOwner(o).size());
```
Comment (wottop) [inline server/src/test/java/org/candlepin/controller/PoolManagerFunctionalTest.java]: why not use pools.size() ?

Comment (nguyenfilip) [inline server/src/test/java/org/candlepin/controller/PoolManagerFunctionalTest.java]: Fixed

**Comment by wottop** [comment]
ack

**Pull request merged**